### PR TITLE
Update members.md

### PR DESCRIPTION
**Pull request by AlbertLee125**
Added Albert Lee Profile, Pictures will be added later.

### CODE REVIEW

**Comment by AlbertLee125** [comment]
I added my profile image and the image is below the profile image so that it is able to distinguish Zedong and me.

**Comment by ZedongPeng** [comment]
The layout is not correct.
You can test it locally following this.
https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll

image

File: members.md [hunk 1]
```diff
@@ -79,16 +79,29 @@ nav-menu: true
...
-<!-- <section id="three">
+<section id="three" class="spotlights">
 	<div class="inner">
 		<header class="major">
-			<h2>Massa libero</h2>
-		</header>
-		<p>Nullam et orci eu lorem consequat tincidunt vivamus et sagittis libero. Mauris aliquet magna magna sed nunc rhoncus pharetra. Pellentesque condimentum sem. In efficitur ligula tate urna. Maecenas laoreet massa vel lacinia pellentesque lorem ipsum dolor. Nullam et orci eu lorem consequat tincidunt. Vivamus et sagittis libero. Mauris aliquet magna magna sed nunc rhoncus amet pharetra et feugiat tempus.</p>
-		<ul class="actions">
-			<li><a href="generic.html" class="button next">Get Started</a></li>
+			<h2>Ph.D</h2>
```
Comment (ZedongPeng) [inline members.md:85]: I think you should move this outside of the section and define it as section one.

**Review by AlbertLee125 — COMMENTED** [review]
Then the outline will be fine. I see.

**Review by AlbertLee125 — COMMENTED** [review]
Then the outline will be fine. I see.

**Comment by bernalde** [comment]
@ZedongPeng Can you make the changes to show Albert how to fix this?

**Pull request merged**